groupie/src/main/java/com/genius/groupie/Item.java

```diff
@@ -93,13 +78,11 @@ public void bind(T viewBinding, int position, List<Object> payloads) {
         return this == item ? 0 : -1;
     }
 
-    public boolean isClickable() {
```

Comment:
I'd like to see this stay, as it doesn't matter unless the click listener is set.